pkg/lib/config/authentication.go

```diff
@@ -97,6 +97,46 @@ var _ = Schema.Add("AuthenticationConfig", `
 				},
 				"required": ["identities"]
 			}
+		},
+		{
+			"if": {
+				"properties": {
+					"identities": {
+						"contains": {
+							"const": "siwe"
+						}
+					}
+				},
+				"required": ["identities"]
+			},
+			"then": {
+				"properties": {
+					"primary_authenticators": {
+						"maxItems": 0
+					}
+				},
+				"required": ["primary_authenticators"]
+			}
+		},
+		{
+			"if": {
+				"properties": {
+					"identities": {
+						"contains": {
+							"const": "siwe"
+						}
+					}
+				},
+				"required": ["identities"]
+			},
+			"then": {
+				"properties": {
+					"identities": {
+						"maxItems": 1
+					}
+				},
+				"required": ["identities"]
+			}
```

Comment:
We can relax the validation on primary_authenticators. Instead, we only need to ensure when `siwe` is in `identities`, `identities` must be `["siwe"]`. That is, siwe and other identity types are mutually exclusive.